snap/hooks/configure

```diff
@@ -40,6 +40,10 @@ if [[ -z $(snapctl get edge-core.refresh-timeout) ]]; then
     snapctl set edge-core.refresh-timeout=300
 fi
 
+
```

Comment:
can you move this up to line 39 to group this with the other kubelet options on lines 31-37?

Reply:
Done